### DISCUSSION TITLE
Change company profile logo

### DISCUSSION THREAD

**Discussion by greenheartcbdpr** [community/community#39425] [New to GitHub]
Hello! We got redesign updated, how we update it here on github? Here is our previous logo: https://github.com/trustwallet/assets/commit/7381a212a3d0eca4cd4d55155e4778a96ed2f415 but now we have new one, it is already updated everywhere

**Comment by BREN0sx** [comment]
Hello! To update your github account profile picture, just **click on your avatar**, or in the top left corner, **settings**, **public profile** and **clicking on your avatar** you will be able to upload or remove the image.
